src/vnodeTransformers/stubComponentsTransformer.ts

```diff
@@ -230,7 +243,8 @@ export function createStubComponentsTransformer({
       return (
         config.plugins.createStubs?.({
           name: stubName,
-          component: type
+          component: type,
+          registerStub
```

Comment:
We want to provide an ability to mess with stubs (including async componentS) to the plugin system, so our plugins will be as capable as original `createStub`